README.md

```diff
@@ -1,54 +1,210 @@
+CTP_API_URL=https://api.commercetools.com or https://api.commercetools.co
+CTP_SCOPES=<your desired scopes>
+```
+
+[1] To read more about the differences between the usage of .env and .env.local you can
+continue [here](https://symfony.com/doc/current/configuration/dot-env-changes.html) and/or
+[here](https://symfony.com/blog/new-in-symfony-4-2-define-env-vars-per-environment)
+
+### Verify everything works
+
+To verify that your configuration works, after adding your client credentials on the .env file
+as explained above, you may run on the command line:
+`bin/console commercetools:project-info`. If everything is set up correctly, this should return
+the details regarding your project like:
+
+```
+Project's key: super-cool-project-4
+Project's name: my eshop at commercetools
+Countries: DE, US
+Currencies: EUR
+Languages: en
+Created at: 2019-02-04T11:28:49+00:00
+Messages: disabled
+```
+
+if so, it means that your project can now communicate with commercetools platform via the client
+you just configured.
+
+## Usage
+
+commercetools symfony bundle consists of 8 smaller bundles (The dependencies mentioned are
+only the additional ones, required for bundle-specific functionalities)
+
+- CartBundle
+- CatalogBundle
+- CtpBundle
+- CustomerBundle
+    - depends: `symfony/security-bundle`
+- ReviewBundle
+- SetupBundle
+    - depends: `symfony/console`
+- ShoppingListBundle
+- StateBundle
+    - depends: `symfony/console`, `symfony/workflow`, `twig/extensions`
+    
+
+By default, the enabled bundles are `CtpBundle` & `CustomerBundle` for all environments, and
+additionally `SetupBundle` & `StateBundle` only for dev environment. To overview, 
+enable or disable  them, you need to edit the file `config/bundles.php` 
+
+
+### Usage of available services 
+
+In every bundle there are services that can be autowired directly in your application. For example
+to autowire `CatalogManager` in a controller action: 
 
 ```php
-class AppKernel extends Kernel
+<?php
+namespace App\Controller;
+
+use Commercetools\Symfony\CatalogBundle\Manager\CatalogManager;
+use Symfony\Component\HttpFoundation\Response;
+
+class MyController
 {
-    public function registerBundles()
+    public function index(CatalogManager $manager)
     {
-        $bundles = [
-            // ...
-            new Commercetools\Symfony\CtpBundle\CtpBundle(),
-            // ...
-        ];
-        // ...
+        /** @var CategoryCollection $categories */
+        $categories = $manager->getCategories('en');
+
+        return new Response(
+            '<html><body>categories: '
+            // handle CategoryCollection $categories
+            .'</body></html>'
+        );
     }
-    // ...
+}
 ```
 
-Open App/config/parameters.yml.dist and add these lines
-```yaml	
-    commercetools.client_id: ~
-    commercetools.client_secret: ~
-    commercetools.project: ~
-```
+The main idea behind is that in each Bundle there are some reusable services that you
```

Comment:
Can you move this entire paragraph up directly above the header?

Reply:
I moved it under  "### Available services",  is that what you meant?